### PR TITLE
upgrade: add upgrade procedure for Wazo 18.03

### PR DESCRIPTION
**Pull request by sduthil**

### CODE REVIEW

**Comment by sboily** [comment]
Can we explain there is some feature are not supported? For example meetme is not supported anymore and there is no migration included to confbridge. Wazo client is not supported anymore and it don't work, the old interface orange has been completely removed etc ... There is a community proc for meetme  to migrate to confbridge here: https://wazo-platform.discourse.group/t/mise-a-jour-de-18-03-vers-20-x/355/9

**Comment by sboily** [comment]
Hello, what is the status?

**Comment by sduthil** [comment]
The status is: waiting for somebody (should be me) to have some time to take comments into account.

File: content/uc-doc/upgrade/upgrade_from_wazo_18.03.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: Upgrading from Wazo 18.03
+---
+
+## Rationale
+
+Wazo 18.03 was the last version before a big gap in Wazo versions: the next
+available version is Wazo 19.13. As a consequence, a lot of Wazo installation
+have been installed in this version Wazo 18.03. Given the big gap of versions,
```
Comment (fblackburn1) [inline content/uc-doc/upgrade/upgrade_from_wazo_18.03.md:9]:
```suggestion
have been installed in this version. Given the big gap of versions,
```

File: content/uc-doc/upgrade/upgrade_from_wazo_18.03.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+title: Upgrading from Wazo 18.03
+---
+
+## Rationale
+
+Wazo 18.03 was the last version before a big gap in Wazo versions: the next
+available version is Wazo 19.13. As a consequence, a lot of Wazo installation
```
Comment (fblackburn1) [inline content/uc-doc/upgrade/upgrade_from_wazo_18.03.md:8]: Upgrading (without wazo-dist-upgrade) will go to 19.12
```suggestion
available version is Wazo 19.12. As a consequence, a lot of Wazo installation
```

**Review by fblackburn1 — CHANGES_REQUESTED** [review]

File: content/uc-doc/upgrade/upgrade_from_wazo_18.03.md (hunk 1 above)
Comment (sduthil) [reply content/uc-doc/upgrade/upgrade_from_wazo_18.03.md]: Fixed, thanks.

File: content/uc-doc/upgrade/upgrade_from_wazo_18.03.md (hunk 2 above)
Comment (sduthil) [reply content/uc-doc/upgrade/upgrade_from_wazo_18.03.md]: Fixed, thanks.

**Review by fblackburn1 — APPROVED** [review]

**Pull request merged**